rosdep/base.yaml

```
@@ -352,6 +352,7 @@ bullet:
   opensuse: [libbullet]
   rhel:
     '7': [bullet-devel]
+    '8': [bullet-devel]
```

We're only targeting RHEL 7 and 8, so the rules can be collapsed in to a single rule.

```suggestion
  rhel: [bullet-devel]
```